SonarQube.Client/Models/ServerExclusions.cs

```diff
@@ -57,5 +57,23 @@ public override string ToString()
                    "\n    Exclusions: " + string.Join(",", Exclusions) +
                    "\n    Global Exclusions: " + string.Join(",", GlobalExclusions);
         }
+
+        public override bool Equals(object obj)
+        {
+            return Equals((ServerExclusions)obj);
```

Comment:
This needs be a soft cast, otherwise this will throw if the object isn't a `ServerExclusions` instance.

Reply:
Just failed on this... Fixed now, my bad.